__tests__/__integration__/imperative/__tests__/__integration__/cli/config/init/cli.imperative-test-cli.config.init.integration.test.ts

```diff
@@ -0,0 +1,381 @@
+/*
+* This program and the accompanying materials are made available under the terms of the
+* Eclipse Public License v2.0 which accompanies this distribution, and is available at
+* https://www.eclipse.org/legal/epl-v20.html
+*
+* SPDX-License-Identifier: EPL-2.0
+*
+* Copyright Contributors to the Zowe Project.
+*
+*/
+
+import { ITestEnvironment } from "../../../../../../../__src__/environment/doc/response/ITestEnvironment";
+import { SetupTestEnvironment } from "../../../../../../../__src__/environment/SetupTestEnvironment";
+import { runCliScript } from "../../../../../../../src/TestUtil";
+import { IConfig } from "../../../../../../../../packages/config";
+import * as fs from "fs";
+import * as path from "path";
+
+
+// Test Environment populated in the beforeAll();
+let TEST_ENVIRONMENT: ITestEnvironment;
+
+describe("imperative-test-cli config init", () => {
+    const expectedSchemaObject = {
```

Comment:
Wondering if this object could be shared amongst different test files?
Trying to avoid the old DEFAULT_PARAMETERS objects we used to have (or still have maybe?) for command handler testing

Reply:
It would probably make a good bit of sense to share that between test files.